back/app/mailers/application_mailer.rb

```diff
@@ -125,6 +132,10 @@ def reply_to_email
     app_settings.core.reply_to_email.presence || ENV.fetch('DEFAULT_FROM_EMAIL')
   end
 
+  def domain
+    custom_from_email&.split('@')&.last
+  end
```

Comment:
Not sure that's needed, but this will fail in case there is no custom_from_email, right?

Either you make this
`from_email.split('@')&.last`

or you change the `default_config` hash to not include the `domain` key in case it's nil.

Reply:
It will return nil, no? It works fine with smtp when there's no custom_from_email. But you could well be right 🤔